src/lib/components/wizard-navigation-bar.component.html

```diff
@@ -1,20 +1,16 @@
 <ul class="steps-indicator steps-{{numberOfWizardSteps}}">
   <li *ngFor="let step of wizardSteps"
-      [ngClass]="{
-        default: isDefault(step),
-        current: isCurrent(step),
-        done: isDone(step),
-        editing: isEditing(step),
-        optional: isOptional(step),
-        navigable: isNavigable(step)
-  }">
+      [ngClass]="getStepClass(step)">
```

Comment:
This way I can add custom class

Reply:
I like the idea of moving the json object creation to a typescript method.
But I don't understand, why a custom class needs to be added via `customNavBarClass`.
Starting with version `4.0.0` of `angular-archwizard`, the intended way to create a default navbar indicator is via the `awWizardStepSymbol` directive.
For an example see https://github.com/madoar/angular-archwizard-demo/blob/develop/src/app/custom-step-symbol-template/custom-step-symbol-template.component.html.